src/python/sense/client/serviceclient.py

```diff
@@ -1,63 +1,38 @@
-import json
-import requests
-from sense.common import evalInput
-from sense.client.mainclient import MainClient
-
-class ServiceClient(MainClient):
-    def __init__(self):
-        super(DriverClient, self).__init__()
-
-    def get_service(self, tags):
-        url = self.config['REST_API'] + tags
-        out = requests.get(url, headers=self.config['headers'], verify=self.config['verify'], auth=(self.config['CLIENT_ID'], self.config['SECRET']))
-        return evalInput(out.text)
-
-    def put_service(self, tags):
-        url = self.config['REST_API'] + tags
-        out = requests.put(url, headers=self.config['headers'], verify=self.config['verify'], auth=(self.config['CLIENT_ID'], self.config['SECRET']))
-        return evalInput(out.text)
-
-    def post_service(self, intent, tags):
-        url = self.config['REST_API'] + tags
-        out = requests.post(url, headers=self.config['headers'], verify=self.config['verify'], data = intent, auth=(self.config['CLIENT_ID'], self.config['SECRET']))
-        return evalInput(out.text)
-
-    def delete_service(self, tags):
-        url = self.config['REST_API'] + tags
-        out = requests.delete(url, headers=self.config['headers'], verify=self.config['verify'], auth=(self.config['CLIENT_ID'], self.config['SECRET']))
-        return evalInput(out.text)
-
-   def commit(self, uuid):
+from sense.client.requestwrapper import RequestWrapper
+class ServiceClient(RequestWrapper):
+   def __init__(self):
+      super(ServiceClient, self).__init__()
+
+   def commit(self, uuid, **kwargs):

```

Comment:
I see an issue if we use short function names, like commit, create, delete. Once more APIs will be added, it might conflict with the same names
We should have the full name of the function, like service_commit, service_create, service_delete... What do you think?